api_v3/services/BaseEntryService.php

```diff
@@ -84,9 +88,27 @@ function addAction(KalturaBaseEntry $entry, $type = -1)
     	myNotificationMgr::createNotification(kNotificationJobData::NOTIFICATION_TYPE_ENTRY_ADD, $dbEntry, $dbEntry->getPartnerId(), null, null, null, $dbEntry->getId());
     	
 	    $entry->fromObject($dbEntry, $this->getResponseProfile());
+		$this->handleMultiLanguageMapping($entry);
```

Comment:
you need to add the handling for multilang in doFromObject else you will have it supported only on some of the apis